spec/models/user_spec.rb

```diff
@@ -86,4 +86,26 @@
       expect(@user.translated_videos).not_to include @video
     end
   end
+
+  describe "leaderboard and points tests" do
+    before(:each) do
+      @user = FactoryGirl.create(:user)
+      @year_t, @month_t, @old_t = Translation.new, Translation.new, Translation.new
+      @year_t.status, @month_t.status, @old_t.status = 2, 1, 1
```

Comment:
Perhaps you should use stubs instead of directly modifying the status columns. This creates a certain level of abstraction.